docs/guide/learning/how-to-learn-vue.md

```diff
@@ -243,6 +243,68 @@ In this section, we will go over the most popular technologies and the struggles
 
 ### Laravel
 
+<!-- TODO: Missing Link -->
+You may be interested in [checking out the dev.to article](). It mirrors this section entirely (heading for heading), going into greater detail, while providing code samples and demos.
```

Comment:
Dont do that, you are pushing people away from the TLDR. Point them to the link after giving then the summary. Most people are interested in mostly that.

Reply:
That makes sense, I'll move it down then!